src/App.js

```diff
@@ -1,19 +1,56 @@
+import { useState } from 'react';
 import React from 'react';
 import './App.css';
+import ChatEntry from './components/ChatEntry';
```

Comment:
Be sure to remove unused imports before opening your PR, accidentally adding in unused imports can make the file cluttered over time.